token-metadata/program/tests/utils/metadata.rs

```diff
@@ -105,7 +105,7 @@ impl Metadata {
         collection: Option<Collection>,
         uses: Option<Uses>,
     ) -> transport::Result<()> {
-        create_mint(context, &self.mint, &context.payer.pubkey(), None).await?;
+        create_mint(context, &self.mint, &context.payer.pubkey(), Some(&context.payer.pubkey())).await?;
```

Comment:
Cool ya I thought this and was thinking maybe it’s fine because looks like all existing tests weren’t using it. But ya makes to be able to specify freeze authority creating a mint so I’m just do that and just not affect those